Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -249,6 +269,7 @@ - (void)handleRemoteNotificationReceived:(NSNotification *)notification
 - (void)handleRemoteNotificationsRegistered:(NSNotification *)notification
 {
   [self sendEventWithName:@"remoteNotificationsRegistered" body:notification.userInfo];
+  [self stopObservingRegisteredNotifications];
```

Comment:
Why do you need to `stopObservingRegisteredNotifications`? IIRC when registering for push (not local) notifications, `RCTRegisterUserNotificationSettings` is called first, followed by `RCTRemoteNotificationsRegistered` if we got push token.

I don't see a reason to stop observing because it adds complexity without much gain.